.github/workflows/phpunit.yml

```
@@ -34,3 +34,28 @@ jobs:
         run: |
           php -v
           ./vendor/bin/phpunit
+
+  build2:
+    name: PHPUnit
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [macos-latest]
```

Seems we need to keep the case, so the build name stays the same

```suggestion
        operating-system: [macOS-latest]
```